Python/testing-sdk-with-prompt-sdk/README.md

```diff
@@ -0,0 +1,259 @@
+Return the results as JSON in the below format:
+
+'''
+{
+  "cards": [
+    {
+      "front": "What is the capital of France?",
+      "back": "Paris"
+    },
+    {
+      "front": "Who painted the Mona Lisa?",
+      "back": "Leonardo da Vinci"
+    }
+  ]
+}
+'''
+
+Only return JSON in your response, nothing else. Do not include the backticks.
+
+Example:
+
+Notes:
+
+'''
+Am. History Notes 🇺🇸
+Beginnings & Stuff
+Columbus 1492, "found" America but actually not the first.
+Native Americans were here first, tons of diff cultures.
+Colonies & Things
+13 Colonies cuz Brits wanted $ and land.
+Taxation w/o Representation = Colonists mad at British taxes, no say in gov.
+Boston Tea Party = Tea in the harbor, major protest.
+Revolution Time
+Declaration of Independence, 1776, basically "we're breaking up with you, Britain".
+George Washington = First pres, war hero.
+Moving West
+Manifest Destiny = Idea that the US was supposed to own all land coast to coast.
+Louisiana Purchase, 1803, Thomas Jefferson bought a ton of land from France.
+'''
+
+Flashcards:
+
+{
+  "cards": [
+    {
+      "front": "Who was the first president of the United States?",
+      "back": "George Washington"
+    },
+    {
+      "front": "What was the idea that the US was supposed to own all land coast to coast?",
+      "back": "Manifest Destiny"
+    },
+    {
+      "front": "What was the year of the Louisiana Purchase?",
+      "back": "1803"
+    }
+  ]
+}
+```
+
+Add a template named `user` and add the following text:
+
+```
+Notes:
+
+'''
+{{ notes }}
+'''
+
+Flashcards:
+```
+
+Deploy your prompt.
+
+### Create your `.autoblocks.yml` file
+
+This file instructs the CLI which prompts and which of their versions
+to autogenerate code for. One has already been created for you in [`./.autoblocks.yml`](./.autoblocks.yml).
+
+### Set your Autoblocks API key
+
+Retrieve your **local testing API key** from the [settings page](https://app.autoblocks.ai/settings/api-keys) and set it as an environment variable:
+
+```bash
+export AUTOBLOCKS_API_KEY=...
+```
+
+### Autogenerate prompt classes
+
+This CLI will autogenerate classes for you to use to interact with the prompts you've created in the UI.
+
+```
+poetry run autoblocks prompts generate
+```
+
+## Run Autoblocks tests
+
+### Set your OpenAI API key
+
+```bash
+export OPENAI_API_KEY=...
+```
+
+### Run the tests
+
+```bash
+npx autoblocks testing exec -m "my first run" -- poetry run start
+```
+
+You should see something like:
+
+<img width="1668" alt="Screenshot 2024-02-22 at 1 23 50 PM" src="https://github.com/autoblocksai/autoblocks-examples/assets/7498009/ded25aa8-1439-432d-86a6-7254b27b970b">
+
+You can click on the links next to each test name to dig into more details.
+You can also find all of your tests on the testing homepage in the [Autoblocks application](https://app.autoblocks.ai/testing/local).
+
+## GitHub Actions setup
```

Comment:
do you wanna drop this? since that action isn't running this script

Reply:
Ya I was debating that. We do a get a lot of questions about CI so I think it's nice to reference it. Maybe instead of this i'll just include a link to the testing docs then that will be updated with CI info

Reply:
oh i see, ok it's fine then